aioredis/commands/streams.py

```diff
@@ -0,0 +1,222 @@
+import sys
+from collections import OrderedDict
+
+from aioredis.util import wait_convert, wait_make_dict, wait_ok
+
+PY_VER = sys.version_info
+
+if PY_VER < (3, 0):
+    from itertools import izip as zip
```

Comment:
These lines don't make sense. We don't support Python 2.x.

Reply:
Yep, not sure what I was thinking here